### PR TITLE
Update Digital Ocean program status

### PR DESCRIPTION
**Pull request by krissnawat**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Dev Spotlight](https://www.devspotlight.com/jobs/) - Around $300-$500 per piece depending on length and content
   > Technical content production agency that works with many clients.
 
-- [Digital Ocean](https://www.digitalocean.com/write-for-donations/) - Up to $400 per piece
+- [Digital Ocean](https://www.digitalocean.com/write-for-donations/) - Up to $400 per piece ****Currently paused accept new writer**
```
Comment (malgamves) [inline README.md:31]: Hey @krissnawat thanks for the addition. Could you change the text to 

`** *Applications currently on hold**`

**Review by malgamves — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LindaOjo) [inline README.md:31]:
```suggestion
- [Digital Ocean](https://www.digitalocean.com/write-for-donations/) - Up to $400 per piece **Applications currently on hold**
```

**Review by LindaOjo — COMMENTED** [review]
I noticed this is a bit delayed, so I went ahead to make the change.

**Comment by malgamves** [comment]
Thanks @LindaOjo - merging 🚀

**Pull request merged**